packages/block-library/src/navigation/edit/use-create-navigation-menu.js

```diff
@@ -25,7 +25,7 @@ export default function useCreateNavigationMenu( clientId ) {
 			const record = {
 				title,
 				content: serialize( blocks ),
-				status: 'publish',
+				status: 'draft',
```

Comment:
This seems like a bit of a drastic change this late. I don't think we should do this.

Reply:
Agreed.

Reply:
I've removed this change.